exercises/scale-generator/canonical-data.json

```diff
@@ -0,0 +1,181 @@
+{
+  "exercise": "scale-generator",
+  "version": "1.0.0",
+  "cases": [
+    {
+      "comments": [
+        "These tests have no interval.",
+        "Chromatic scale is considered the default scale"
```

Comment:
Very small nit: "Chromatic scale is considered the default scale" should probably be "The chromatic scale is considered the default scale."

Reply:
Nit has been picked